using_images/other_images/jenkins.adoc

```diff
@@ -362,22 +362,24 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
 plug-in] that allows Jenkins slaves to be dynamically provisioned on multiple
 container hosts using Kubernetes and {product-title}.
 
-To facilitate the using of the Kubernetes plug-in, {product-title} provides three images
-suitable for use as Jenkins slaves..
+To use the Kubernetes plug-in, {product-title} provides three images
+suitable for use as Jenkins slaves; the *_Base_*, *_Maven_*, and *_NodeJS_* images.
 
 The first is a https://github.com/openshift/jenkins/tree/master/slave-base[base image] for Jenkins slaves.
```

Comment:
As the nest little section is a list, perhaps end the sentence above with a colon as well.